Scoop/Controller/PostRideLocationViewController.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  PostRideLocationViewController.swift
+//  Scoop
+//
+//  Created by Elvis Marcelo on 3/9/22.
+//
+
+import UIKit
+
+class PostRideLocationViewController: UIViewController {
+    
+    private let methodLabel = UILabel()
+    private let transField = UITextField()
+//    private var transLabel = UIImage()
+    
+    private let locationLabel = UILabel()
+    
+    private let departureField = UITextField()
+    private let arrivalField = UITextField()
+
+    override func viewDidLoad() {
+        
+        self.navigationItem.title = "Trip Details"
+        view.backgroundColor = .white
+        
+        methodLabel.text = "Method of Transportation"
+        methodLabel.textColor = .black
+        view.addSubview(methodLabel)
+        
+        transField.placeholder = "select"
+        transField.textColor = .black
+        view.addSubview(transField)
+        
+        let transImage = UIImage(systemName: "location")
+        let transView = UIImageView(image: transImage)
+        transView.tintColor = .black
+        transView.frame = CGRect(x: 0, y: 0, width: 20, height: 20)
+        view.addSubview(transView)
+        
+        let destImage = UIImage(systemName: "mappin.and.ellipse")
+        let destView = UIImageView(image: destImage)
+        destView.tintColor = .black
+        destView.frame = CGRect(x: 0, y: 0, width: 20, height: 20)
+        view.addSubview(destView)
+        
+        locationLabel.text = "Locations"
+        locationLabel.textColor = .black
+        view.addSubview(locationLabel)
+
+        departureField.placeholder = "Departure Location..."
+        departureField.textColor = .black
+        view.addSubview(departureField)
+
+        arrivalField.placeholder = "Arrival Location..."
+        arrivalField.textColor = .black
+        view.addSubview(arrivalField)
+        
+        methodLabel.snp.makeConstraints { make in
+            make.top.equalToSuperview().offset(300)
+            make.leading.equalToSuperview().offset(80)
```

Comment:
Change to inset(60)

Reply:
You're also never setting the trailing anchor. This should be set also be set to inset(60)